src/geotop/input.cc

```diff
@@ -346,6 +347,8 @@ void get_all_input(long  /*argc*/, char * /*argv*/[], TOPO *top, SOIL *sl, LAND
     success = read_meteostations_file(met->imeteo_stations.get(), met->st.get(), files[fmetstlist],
                                       IT->meteostations_col_names);
 
+#pragma omp parallel for firstprivate(added_JDfrom0, added_wind_xy, added_wind_dir, added_Tdew, added_RH, added_Pint) private(f, i, j, n, ist, temp, num_lines)
```

Comment:
I didn't look at the function implementations but those variables are just read? if so, please make them shared. there is no need to create multiple copies

Reply:
which variables? If I don't wite a type explicitly, for OpenMP all variables are shared by default (Ivan Girotto told it during lectures). I've put  some variables **firstprivate** since they are initialize before the parallel region and they will be modified inside (i.e, ``` added_JDfrom0 = fixing_dates(ist, met->data[i-1], par->ST, (*met->st->ST)(i),
                                         met->numlines[i-1], iDate12, iJDfrom0);```)

Reply:
I confirm that  if a variable is  "**shared**", both declared it explicitely or not say anything about it, no copies should be created .